src/graph_ir/tests/test_low_graph_ir.cc

```diff
@@ -0,0 +1,21 @@
+#include "low_graph_ir.h"
+
+#include <vector>
+#include <string>
+#include <iostream>
+
+namespace raksha_graph_ir {
+
+int main() {
+    LowGraphAccessPath* p_a = new LowGraphAccessPath("a");
```

Comment:
If you use `new` to construct something, it is best to `delete` it later. But in this case, where all of these items are used locally in the same scope, it's better to not get the heap involved at all. Just declare these objects in automatic storage like so:

```C++
LowGraphAccessPath p_a("a");
```